packages/bot-web-ui/src/components/load-modal/recent-workspace.tsx

```diff
@@ -20,6 +20,7 @@ const RecentWorkspace = observer(({ workspace }: TRecentWorkspaceProps) => {
             })}
             key={workspace.id}
             onClick={selected_strategy_id === workspace.id ? undefined : () => previewRecentStrategy(workspace.id)}
+            data-testid='recent-workspace-item'
```

Comment:
@shafin-deriv in deriv app for data-testid , i guess we name starting with `dt_` . for example here the data-testid can be `dt_recent_workspace_item`

Reply:
Sure Bhai